test/integration/query-interface/changeColumn.test.js

```diff
@@ -181,6 +181,47 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
           expect(newForeignKeys).to.be.an('array');
           expect(newForeignKeys).to.have.lengthOf(1);
           expect(newForeignKeys[0].columnName).to.be.equal('level_id');
+
+          if (dialect === 'postgres') {
+            const updatedForeignKeys = await this.sequelize.query(
+              this.queryInterface.queryGenerator.getForeignKeysQuery('users', this.sequelize.config.database),
+              { type: this.sequelize.QueryTypes.FOREIGNKEYS },
+            );
+            expect(Object.keys(updatedForeignKeys[0])).to.have.length(8);
+            expect(updatedForeignKeys[0].condef).to.equal('FOREIGN KEY (level_id) REFERENCES level(id) ON UPDATE CASCADE ON DELETE CASCADE');
+          }
```

Comment:
same idea as previous comment here.

I think a more interesting series of test for `changeColumn` would be:

- column has `NOT VALID`, and `changeColumn` has `notValid` set to false (removes it)
- column does not have `NOT VALID`, and `changeColumn` has `notValid` set to true (adds it)
- column has `NOT VALID`, and `changeColumn` has `notValid` not set or undefined (leaves it unchanged, not removed)